core/src/allocation_init.jl

```diff
@@ -405,7 +405,10 @@ function add_constraints_conservation_node!(
     end
 
     # Only the node IDs with conservation constraints on them
-    node_ids = keys(inflows)
+    node_ids = [
+        node_id for node_id in keys(inflows) if
+        !(isempty(inflows[node_id]) && isempty(outflows[node_id]))
+    ]
```

Comment:
Can you explain this change? Perhaps as a comment.

Reply:
It makes no sense to create a constrant `0 == 0`, so if there are no terms on either side of the equation, no not make the constraint.